transports/dns/src/lib.rs

```diff
@@ -385,7 +327,7 @@ where
     }
 }
 
-/// The possible errors of a [`GenDnsConfig`] wrapped transport.
+/// The possible errors of a [`Config`] wrapped transport.
 #[derive(Debug)]
 #[allow(clippy::large_enum_variant)]
 pub enum DnsErr<TErr> {
```

Comment:
Can we rename this one to just `Error`? (With a type-alias as well)

Reply:
I got it.